structlog_sentry/__init__.py

```diff
@@ -168,6 +168,21 @@ def _handle_breadcrumb(self, event_dict: EventDict) -> None:
             event, hint = self._get_breadcrumb_and_hint(event_dict)
             self._get_hub().add_breadcrumb(event, hint=hint)
 
+    @staticmethod
+    def _get_level_value(level_name: str) -> int:
+        """Get numeric value for the log level name given."""
+        try:
+            # Try to get one of predefined log levels
+            return getattr(logging, level_name)
```

Comment:
Wouldn't this fix the issue you described?

```suggestion
            return getattr(logging, level_name.upper())
```

Reply:
Aha, I see, thank you for explaining it. I think I can approve this and will create a separate issue for this. I just have to think how to fix that.